ci-operator/step-registry/acm/policies/openshift-plus/acm-policies-openshift-plus-commands.sh

```
@@ -18,19 +18,28 @@ echo 'y' | ./deploy.sh -p policygenerator/policy-sets/stable/openshift-plus -n p
+    echo "Try ${try}/${RETRIES}: No policy rows found. Checking again in 30 seconds"
+    sleep 30
+    continue
+  fi
   notready=$(echo "$results" | grep -E 'NonCompliant|Pending' || true)
-  if [ "$notready" == "" ]; then
+  unevaluated=$(echo "$data_rows" | grep -v 'Compliant' | grep -vE "$SECONDARY_POLICIES" || true)
+  if [ "$notready" == "" ] && [ "$unevaluated" == "" ]; then
     echo "OPP policyset is applied and compliant"
     break
   else
     if [ $try == $RETRIES ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)